scripts/echo_crds_context

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+if [[ "${CRDS_CONTEXT}" ]]; then
+    echo $(crds list --contexts ${CRDS_CONTEXT} --resolve-contexts)
+else
+    crds list --operational-context
+fi
```

Comment:
This script gives a consistent way to print out the current CRDS_CONTEXT pmap, regardless of how it is set.  Observe usage in where context is not set (and gets the operational context), set to the floating "jwst-edit" or set to a specific context.

```
$ unset CRDS_CONTEXT
$ echo_crds_context 
jwst_0672.pmap
$ export CRDS_CONTEXT=jwst-edit
$ echo_crds_context 
jwst_0674.pmap
$ export CRDS_CONTEXT=jwst_0673.pmap
$ echo_crds_context 
jwst_0673.pmap
$ unset CRDS_CONTEXT
$ echo_crds_context 
jwst_0672.pmap
```